privacyidea/static/components/config/views/config.events.list.html

```diff
@@ -78,7 +78,7 @@
             <td>
                 <button class="btn btn-danger"
                         ng-show="checkRight('eventhandling_write')"
-                        ng-click="delEvent(event.id)">
```

Comment:
see above

Reply:
fixed in 9c208ab4212d0e7fd5a0133faf6722115b3a40c9.